### DISCUSSION TITLE
Selection of sBTC Signer Set

### DISCUSSION THREAD

**Discussion by andrerserrano** [stacks-sbtc/sbtc#624] [Design]
# Selection of sBTC Signer Set

**Overview**
This discussion aims to outline the process for selecting the initial signer set for the sBTC protocol, in alignment with the requirements specified in [SIP-028](https://github.com/stacksgov/sips/pull/186#pullrequestreview-2194954687). The goal is to gather feedback on the proposed criteria, the selection process, and final signer set to ensure the reliability and decentralization of the protocol.

**Background**
sBTC Signers facilitate the flow of deposits and withdrawals into the sBTC protocol. Signers are responsible for operating the sBTC bridge and ensuring a secure environment for sBTC users. The selected entities will play a key role in maintaining the protocol's functionality and reliable operations. More detailed background information can be found in this [document](https://bitcoinl2labs.notion.site/sBTC-Signer-Brief-10b93bf8305d8009b47ec24fa370f777). 

**Signer Criteria**
The following are the proposed evaluation criteria for selecting the sBTC signers:

- Does the proposed sBTC signer have a demonstrable operating history which shows their experience and reliability in running blockchain services?
- Has the proposed sBTC signer participated in running a Stacks signer instance on Stacks 2.5 testnet or mainnet, and can they provide metrics showing this (ex: amount of stacks stacked over past several cycles)?
- Does the proposed sBTC signer agree to use reasonable efforts to maintain >99% uptime on the sBTC Signer?  (Note: This metric may be self-affirmed if independent verification is not possible, or confirmed by on-chain voting/stacking activity.)
- Does the proposed sBTC signer commit to a direct communication channel to be set up with the sBTC core engineers in order to respond to urgent updates within 24 hours?
- Has the signer made contributions to Bitcoin or the Stacks network over the past year that demonstrate their commitment to the growth and success of the network? (Examples include, but are not limited to: publishing independent research, marketing, co-authoring a SIP, submitting a Stacks pull request/issue, providing feedback on Stacks core development, or contributing to a Stacks Working Group.)

**Selection Process**
The proposed selection process is as follows:

1. **Nomination Phase**: Open a call for nominations within the community. We have published this [Blocksurvey](https://blocksurvey.io/sbtc-signers-7y4TghwsSGqIF1X8m0Cqbg?v=o) to collect Signer nominations. 
2. **Evaluation & Community Feedback**: The proposed signer set will be published to ensure transparency and consensus.
3. **SIP Vote**: The community will vote on the signer criteria. 
4. **Final Selection**: The final list of signers will be selected based on successful completion of [SIP-028](https://github.com/stacksgov/sips/pull/186#pullrequestreview-2194954687).

**Approximate Timeline (Projected)**
- **Nomination Phase**: October 1 - 10
- **Evaluation & Community Feedback**: October 10 - 27
- **SIP Vote**: October 28 - November 10
- **Final Selection**: November 15 (Assuming successful SIP vote)

**Proposed sBTC Signers**
To be completed following the end of the Nomination Phase.

**Comment by andrerserrano** [comment]
At the end of the nomination period, the sBTC Working Group received 22 signer nominations. The signers were evaluated based on the selection criteria and the list was narrowed down to the following 15 signers. Given there were more qualified nominations than available spots, each applicant was placed into categories comprising institutional, ecosystem, and emergent partners. Then, signers we're selected from each category to ensure the signer set represented a diverse cohort of contributors. 

**Proposed sBTC Signer Set**


| **Company Name**      | **Qualifications**                                                                                                                                                                 | **Strengths**                                                                                      |
|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
| **ALUM Labs**          | Institutional-grade infrastructure, reliable STX signer, committed to sBTC launch support.                                                                                        | 99% SLA commitment, high responsiveness, flexibility.                                                |
| **Asymmetric Research**| Top validator on various networks (Solana, Aptos), providing critical feedback to Stacks ecosystem.                                                                                | High-performance infrastructure, security expertise.                                                 |
| **Bitcoin L2 Labs**    | Core development company for sBTC protocol, deep involvement in Stacks and Bitcoin ecosystems.                                                                                    | Extensive knowledge of sBTC protocol, long-standing contributions to Stacks and Bitcoin.             |
| **Blockdaemon**        | Largest node operator, 50+ blockchains supported, active as a signer in Stacks.                                                                                                   | Technical feedback, operational reliability, broad blockchain experience.                            |
| **Chorus One**         | Supports 60+ networks, Chorus One SDK development, active on Stacks mainnet and testnet.                                                                                          | 99.8% uptime guarantee, strong ecosystem contributions.                                              |
| **Degen Lab**          | Continuous involvement with the Stacks ecosystem, successfully built a decentralized stacking pool, and contributor to several open-source projects                                                                                             | Long-term commitment to the ecosystem, high responsiveness, strong stacking experience.                                              |
| **DeSpread**           | Validator for multiple networks, strong presence in Asia, contributing to Stacks decentralization.                                                                                | Validator experience, significant contributions to decentralization and tool development, Stacks regional partner.            |
| **Fast Pool**          | Oldest Stacking pool, deep knowledge of Bitcoin and Stacks nodes since 2021.                                                                                                     | Long operational history, contributions to sBTC code.                                                |
| **Kiln**               | Manages $8.6 billion in staked assets, SOC2 Type 2 certified, track record on Stacks network.                                                                                            | Deep expertise in Bitcoin and PoS protocols, proven security standards.                                 |
| **Luganodes**          | Institutional staking provider, manages $2 billion in assets, ISO27001, SOC2 Type 2 certified.                                                                                   | Trusted by financial institutions, strong focus on compliance and infrastructure security.           |
| **SenseiNode**         | Top Latin American node operator, 9,500+ validators, focused on decentralizing infrastructure.                                                                                    | Extensive validator experience, strong presence in LATAM.                                            |
| **Stacking DAO**       | Leading DeFi app and liquid staking protocol, managing 56 million STX, sBTC signer infrastructure.                                                                               | Largest validator network in Stacks, focus on decentralization.                                      |
| **Restake**            | Institutional blockchain infrastructure provider, spin-off from one of Europe’s largest bitcoin miners. One of the first signers on Stacks and backer of Stacks ecosystem projects.                                                                          | Strong infrastructure, security practices, active ecosystem contributions.                           |
| **Xverse**             | Long-term Stacks supporter, largest Stacking pool by TVL, integrated into Xverse wallet.                                                                                         | Deep integration with Stacks ecosystem, history of operating, strategic sBTC integration.                                           |


**Next Steps**
The finalized list of 15 signers will now enter the feedback period to allow for thorough review of the selected signers. Following this phase, a SIP vote will take place with the final selection being announced in November.

We invite the Stacks community to participate in this process. Please share any thoughts or questions regarding the criteria for selected signers in this discussion thread. 

**Edit**: As core contributors make final preparations for the launch, 14 of these initial 15 Signers are ready to go. While one Signer (Figment) was unable to move forward, this possibility was accounted for in both the design of the protocol and the SIP. The network will move forward with 14 elected Signers, still requiring the originally intended 70% threshold (or 10/14 Signers) for sBTC transactions. The sBTC Signers will run a vote to replace the last signer early next year.

**Comment by andre-stacks** [comment]
## sBTC Signer Set Update: Adding xLabs, Removing Luganodes

Following the process outlined in SIP-028 for updating the sBTC signer set, we are proposing the following change:

### Addition: [xLabs](https://www.xlabs.xyz/)

xLabs is a core contributor to Wormhole and brings significant experience in cross-chain infrastructure and validator operations. Key qualifications include:

| Criteria | Details |
|----------|---------|
| **Operating History** | Develops and maintains core Wormhole smart contracts, liquidity layers, and composable intents. Operates Wormhole Guardian nodes (testnet and mainnet) and the relayer stack with 24/7 coverage. Builds and maintains Portalbridge.com and WormholeScan. |
| **Infrastructure Experience** | Deployed 100+ RPC nodes across 35+ blockchains. First validator in Argentina and one of the few in South America, contributing to infrastructure decentralization in LATAM. |
| **Testnet Participation** | Active participant in Wormhole's early testnet, with ongoing involvement in testing new features, implementing security fixes, and validating updates. |
| **Uptime Commitment** | ✅ Yes |
| **24-Hour Communication Commitment** | ✅ Yes |
| **Geographic Coverage** | Europe and South America |

xLabs has committed to contributing to the Stacks ecosystem through technical improvements, community engagement, and promoting Stacks adoption - building on their track record with Wormhole and other networks.

### Removal: Luganodes

Luganodes will be removed from the active signer set.
